### ISSUE TITLE
Update example : isAvailable takes two functions as in-parameters

### ISSUE THREAD

**Issue by elsewhat** [EddyVerbruggen/cordova-plugin-safariviewcontroller#7]
Hi, 

As far as I can see the example in the 4.Usage section is not correct. 
It will work on iOS9 devices, but the fallback will not work. 

The example should be updated so that it provides two functions ref the interface defined in SafariViewController.js:

  isAvailable: function (onSuccess, onError) {
    exec(onSuccess, onError, "SafariViewController", "isAvailable", []);
  },

PS huge thanks for providing this plugin!

**Comment by idpaterson** [comment]
I think that providing an error handler would be confusing when the success callback already takes a boolean argument. There are no situations where it would be appropriate for the call to fail rather than succeeding with a `true` or `false` value for the callback param. Perhaps this code would self-document better if the interface were as follows?
```js
isAvailable: function (callback) {
    exec(callback, null, "SafariViewController", "isAvailable", []);
},
```

**Comment by EddyVerbruggen** [comment]
Totally agree - just released 1.1.1 which gets rid of the `onError` handler. :+1:
